### PR TITLE
Update globalgroups.yml for 1.16

### PR DESCRIPTION
**Pull request by super-cokil**
Added 1.16 permissions and updated old permissions

### CODE REVIEW

File: resources/globalgroups.yml [hunk 1]
```diff
@@ -51,48 +51,70 @@ groups:
 
   g:bukkit_moderator:
     permissions:
-    - bukkit.command.ban
-    - bukkit.command.ban.ip
-    - bukkit.command.ban.player
+    - minecraft.command.ban
+    - minecraft.command.ban.ip
+    - minecraft.command.ban.player
```
Comment (l1ttleO) [inline resources/globalgroups.yml:56]:
```suggestion
    - minecraft.command.ban
    - minecraft.command.ban-ip
```

File: resources/globalgroups.yml [hunk 2]
```diff
@@ -51,48 +51,70 @@ groups:
...
+    - minecraft.command.teleport
+    - minecraft.command.time
+    - minecraft.command.weather
+    - minecraft.command.whitelist
+    - minecraft.command.whitelist.add
+    - minecraft.command.whitelist.disable
+    - minecraft.command.whitelist.enable
+    - minecraft.command.whitelist.list
+    - minecraft.command.whitelist.reload
+    - minecraft.command.whitelist.remove
```
Comment (l1ttleO) [inline resources/globalgroups.yml:103]: There's only 1 permission for managing whitelists - `minecraft.command.whitelist`
```suggestion
    - minecraft.command.whitelist
```
```

File: resources/globalgroups.yml [hunk 3]
```diff
@@ -51,48 +51,70 @@ groups:
...
+    - minecraft.command.xp
+    - bukkit.broadcast
+    - bukkit.broadcast.admin
+    - bukkit.command.help
     - bukkit.command.plugins
     - bukkit.command.reload
     - bukkit.command.save
     - bukkit.command.save.disable
     - bukkit.command.save.enable
     - bukkit.command.save.perform
```
Comment (l1ttleO) [inline resources/globalgroups.yml:115]: Saving commands are handled by Vanilla, not Bukkit
```suggestion
    - minecraft.command.save-all
    - minecraft.command.save-on
    - minecraft.command.save-off
```

**Review by l1ttleO — CHANGES_REQUESTED** [review]

**Comment by ElgarL** [comment]
OK, I clearly don't understand Github conversations, they get ignored when you accept the pull request.

**Pull request merged**